gradle/dependencies.gradle

```diff
@@ -13,15 +13,15 @@ ext {
   // version which we should use in this build
   def mapboxNavigatorVersion = System.getenv("FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION")
   if (mapboxNavigatorVersion == null || mapboxNavigatorVersion == '') {
-      mapboxNavigatorVersion = '129.0.0'
+      mapboxNavigatorVersion = '130.0.0'
   }
   println("Navigation Native version: " + mapboxNavigatorVersion)
 
   version = [
-      mapboxMapSdk              : '10.12.0-rc.1',
+      mapboxMapSdk              : '10.12.1',
       mapboxSdkServices         : '6.11.0',
       mapboxNavigator           : "${mapboxNavigatorVersion}",
-      mapboxCommonNative        : '23.4.0-rc.1',
+      mapboxCommonNative        : '23.4.0',
```

Comment:
We should mention in the changelog that the known issue from 2.12.0-beta.3 is fixed.